psimi/psixml/intact-psixml-exchange/pom.xml

```diff
@@ -40,31 +40,38 @@
             </exclusions>
         </dependency>
 
+
+        <dependency>
+            <groupId>uk.ac.ebi.intact.core</groupId>
+            <artifactId>intact-core</artifactId>
+            <version>${core.version}</version>
+        </dependency>
+
         <!--do not update-->
```

Comment:
The dependency on `uk.ac.ebi.intact.dataexchange:intact-enricher-2.1.10` and the dependency on `uk.ac.ebi.intact.bridges:intact-taxonomy-2.1.26` (line 84) could not be updated for `PsiEnricherImpl` to compile.